### PR TITLE
search-block: translate some missing strings to german and fix a typo

### PR DESCRIPTION
**Pull request by pbauer**

### CODE REVIEW

File: locales/de/LC_MESSAGES/volto.po [hunk 1]
```diff
@@ -287,12 +287,12 @@ msgstr "Erweiterung erfolgreich aktualisiert"
...
 # defaultMessage: Advanced facet?
 msgid "Advanced facet?"
-msgstr ""
+msgstr "Erweiterte Facette?"
 
 #: components/manage/Blocks/Search/schema
 # defaultMessage: Advanced facets are initially hidden and displayed on demand
 msgid "Advanced facets are initially hidden and displayed on demand"
-msgstr ""
+msgstr "Erweiterte Facetten sind zunächst verborgen und werden auf Anforderung angezeigt"
```
Comment (ksuess) [inline locales/de/LC_MESSAGES/volto.po:295]:
```suggestion
msgstr "Erweiterte Facetten sind zunächst verborgen und werden auf Anforderung angezeigt."
```

File: locales/de/LC_MESSAGES/volto.po [hunk 2]
```diff
@@ -287,12 +287,12 @@ msgstr "Erweiterung erfolgreich aktualisiert"
 #: components/manage/Blocks/Search/schema
 # defaultMessage: Advanced facet?
 msgid "Advanced facet?"
-msgstr ""
+msgstr "Erweiterte Facette?"
```
Comment (ksuess) [inline locales/de/LC_MESSAGES/volto.po:290]: I suggest to avoid question marks in all options and all languages. 
@stevepiercy what's your opinion?

**Review by ksuess — CHANGES_REQUESTED** [review]
I suggest to change "Facetten-Widget" to "Widget", which is shorter and spares therefore a not so nice line wrap.
#: components/manage/Blocks/Search/schema
# defaultMessage: Facet widget
msgid "Facet widget"
msgstr "Widget"

File: locales/de/LC_MESSAGES/volto.po (hunk 2 above)
Comment (pbauer) [reply locales/de/LC_MESSAGES/volto.po:290]: We already have "Multiple choices?" and "Show sorting?" and other cases. So I'd say that out of scope of this PR.

**Comment by pbauer** [comment]
> Facet widget

I agree, but that's also out of scope

**Review by ksuess — APPROVED** [review]

**Pull request merged**